cmd/agent/subcommands/run/command.go

```diff
@@ -392,7 +392,7 @@ func startAgent(
 
 	// start remote configuration management
 	var configService *remoteconfig.Service
-	if pkgconfig.Datadog.GetBool("remote_configuration.enabled") {
+	if pkgconfig.IsRemoteConfigEnabled() {
```

Comment:
*Naive question*: I noticed that you didn't apply the same change on [cmd/agent/subcommands/remoteconfig/command.go](https://github.com/DataDog/datadog-agent/blob/baptiste.foy/RCM-1079/disable-rc-govcloud/cmd/agent/subcommands/remoteconfig/command.go#L57C1-L58C1).
Shouldn't the remote-config state command return *Remote configuration is not enabled* for FED customers instead of returning the real flag value ?

Reply:
Yep you're right, I forgot about that one. Nice catch! As that file is powered by a component, I updated my method in [b0e8faa](https://github.com/DataDog/datadog-agent/pull/17967/commits/b0e8faa7331955f2e04db6db466a2e608a18ca65) to support a `Config` interface instead.